### PR TITLE
S3 data lake documentation updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added S3 bucket permissions in the S3 data lake documentation
> Repurposed some information for better clarity and flow
> Refined the content and updated image paths

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [x] Hotfix

## Notion ticket link

> [S3 permissions for S3 data lake destination](https://www.notion.so/rudderstacks/S3-permissions-for-S3-data-lake-eaaf307db5bb4fabbb8cac2d14366e4f)

### CODE REVIEW

File: docs/destinations/warehouse-destinations/s3-datalake.mdx [hunk 1]
```diff
@@ -167,41 +191,39 @@ Since all tables are created in the path <code class="inline-code">s3://testBuck
...
 
 <div class="warningBlock">
-Before querying your data on S3, make sure that you have sent some data to S3 and that the sync has been completed.
+Before querying your data on S3, make sure that you have sent some data to S3 and that the sync is completed.
 </div>
 
-Follow these steps to start querying your data on s3 -
+Follow these steps to start querying your data on s3:
 
 1. Open your AWS Athena console. Then, go to the same AWS region which was used while to configure AWS Glue.
```
Comment (achettyiitr) [inline docs/destinations/warehouse-destinations/s3-datalake.mdx:210]: `while to configure` => `while configuring`

**Review by achettyiitr — CHANGES_REQUESTED** [review]

File: docs/destinations/warehouse-destinations/s3-datalake.mdx [hunk 2]
```diff
@@ -17,70 +17,97 @@ Find the open source transformer code for this destination in the <a href="https
...
 
-<img src="../../assets/connectionScreen.png" alt="S3 data lake destination settings in RudderStack" />
+Enter the following settings in the **Connection Credentials** page:
 
-Enter the following credentials in the **Connection Credentials** page:
+<img src="../../assets/dw-integrations/s3-datalake-connection-settings.png" alt="S3 data lake destination settings in RudderStack" />
 
-* **S3 Storage Bucket Name**: The name of the S3 bucket that will be used to store the data before loading it into the S3 data lake. 
+* **S3 Storage Bucket Name**: Enter the name of the S3 bucket used to store the data before loading it into the S3 data lake.
 * **Register schema on AWS Glue**: If you enable this option, RudderStack registers the schema of your incoming data on AWS Glue's data catalog.
```
[the remark this answers is not in the document]
Comment (ameypv-rudder) [reply docs/destinations/warehouse-destinations/s3-datalake.mdx:32]: we use American English, so `catalog` would be more suitable here. [Reference](https://www.grammarly.com/blog/catalog-catalogue/#:~:text=In%20the%20United%20States%2C%20catalog,are%20common%20in%20American%20English.).

File: docs/destinations/warehouse-destinations/s3-datalake.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/destinations/warehouse-destinations/s3-datalake.mdx:210]: thanks for highlighting - this is fixed.

File: docs/destinations/warehouse-destinations/s3-datalake.mdx [hunk 3]
```diff
@@ -17,70 +17,97 @@ Find the open source transformer code for this destination in the <a href="https
...
-<li>glue:UpdateTable</li>
-<li>glue:CreateDatabase</li>
-<li>glue:GetTables</li>
-</ul>
+Make sure the credentials (<strong>Access Key ID</strong> and <strong>Secret Access Key</strong>) have the permissions to read and write into the configured S3 bucket.
 </div>
 
+## S3 permissions for data lake destination
+
+If you have disabled the **Register schema on AWS Glue** setting in the <Link to="#connection-settings">destination settings</Link>, then you need to set the following permissions in your S3 policy to successfully send data to your S3 data lake:
```
Comment (achettyiitr) [inline docs/destinations/warehouse-destinations/s3-datalake.mdx:82]: These are the default permissions. It is irrespective of whether Glue is enabled or not.

File: docs/destinations/warehouse-destinations/s3-datalake.mdx [hunk 4]
```diff
@@ -17,70 +17,97 @@ Find the open source transformer code for this destination in the <a href="https
...
 
 <div class="infoBlock">
 For more information on registering your schema in AWS Glue, refer to the <a href="https://docs.aws.amazon.com/glue/latest/dg/schema-registry.html">AWS Glue documentation</a>.
 </div>
 
-* **AWS Glue Region**: Your AWS Glue region. For example, for `N.Virginia`, it would be `us-east-1`.
-* **S3 Prefix**: If specified, RudderStack creates a folder in the bucket with this prefix and push all the data within that folder. 
-* **Namespace**: If specified, all the data for the destination will be pushed to `s3://<bucketName>/<prefix>/rudder-datalake/<namespace>`. If you don't specify a namespace in the settings, it is set to the source name, by default.
+<div class="warningBlock">
+If AWS Glue is enabled, make sure you grant the following permissions to it:
```
Comment (achettyiitr) [inline docs/destinations/warehouse-destinations/s3-datalake.mdx:39]: If Glue is enabled, then we need to add these extra permissions.

**Review by achettyiitr — CHANGES_REQUESTED** [review]

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/warehouse-destinations/s3-datalake.mdx [hunk 5]
```diff
@@ -3,84 +3,107 @@ title: "Amazon S3 Data Lake"
...
-To send event data to SQL Server, you first need to add it as a destination in RudderStack and connect it to your data source. Once the destination is enabled, events will automatically start flowing to SQL Server via RudderStack.
+```json
+"Action": [
+    "s3:GetObject",
+    "s3:PutObject",
+    "s3:PutObjectAcl",
+    "s3:ListBucket"
+]
+```
+
```
Comment (ameypv-rudder) [inline docs/destinations/warehouse-destinations/s3-datalake.mdx:30]:
```suggestion

<div class="infoBlock">
For detailed steps on creating the IAM user credentials from scratch, refer to the <Link to="/destinations/streaming-destinations/amazon-s3/#permissions">Amazon S3 permissions</Link> section.
</div>
```

**Review by achettyiitr — APPROVED** [review]
LGTM

**Pull request merged**